website/common/locales/en/settings.json

```diff
@@ -16,6 +16,7 @@
     "startAdvCollapsed": "Advanced Options in tasks start collapsed",
     "startAdvCollapsedPop": "With this option set, Advanced Options will be hidden when you first open a task for editing.",
     "dontShowAgain": "Don't show this again",
+    "suppressLoginIncentiveModal": "Don't show popup when checking in",
```

Comment:
@kevinsearle That's a good idea! I think some would like to be able to see the ones where you've earned an item but hide the others.